ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -795,9 +801,13 @@ static ProtocolSpecBuilder pragueDefinition(
 
         // EIP-7002 Withdrawals / EIP-6610 Deposits / EIP-7685 Requests
         .requestsValidator(pragueRequestsValidator(depositContractAddress))
+        .requestsValidator(pragueRequestsValidator(consolidationRequestContractAddress))
         // EIP-7002 Withdrawals / EIP-6610 Deposits / EIP-7685 Requests
         .requestProcessorCoordinator(
-            pragueRequestsProcessors(withdrawalRequestContractAddress, depositContractAddress))
+            pragueRequestsProcessors(
+                withdrawalRequestContractAddress,
+                depositContractAddress,
+                consolidationRequestContractAddress))
```

Comment:
This could use a refactoring. Maybe a new class: `RequestContractAddresses` or something similar

Reply:
> This could use a refactoring. Maybe a new class: `RequestContractAddresses` or something similar

Which of the directories would you suggest I put the new file

Reply:
` org.hyperledger.besu.ethereum.mainnet.requests` should be fine

Reply:
> This could use a refactoring. Maybe a new class: `RequestContractAddresses` or something similar

I have created the class.